### PR TITLE
Add workflow.models.Product

### PR DESCRIPTION
**Pull request by Menda**
## Purpose
Adds a new model connected to WorkflowLevel2

Related ticket: https://github.com/Humanitec/ActivityAPI/issues/105

### CODE REVIEW

File: workflow/models.py [hunk 1]
```diff
@@ -1265,6 +1248,18 @@ def __unicode__(self):
         return unicode(self.workflowlevel1)
 
 
+class Product(models.Model):
+    workflowlevel2 = models.ForeignKey(WorkflowLevel2, related_name="products")
+    name = models.CharField(max_length=255)
+    type = models.CharField(max_length=255, blank=True, null=True)
+    number = models.CharField("Product identifier", max_length=255, blank=True, null=True)
```
Comment (jefmoura) [inline workflow/models.py]: This name sounds too generic. It's better to use something like `external_id`.

Comment (Menda) [reply workflow/models.py]: I was thinking the same. Maybe `reference_id`?

File: workflow/models.py [hunk 2]
```diff
@@ -1143,27 +1135,19 @@ class WorkflowLevel2(models.Model):
     actual_start_date = models.DateTimeField(blank=True, null=True)
     actual_end_date = models.DateTimeField(blank=True, null=True)
     actual_duration = models.CharField(max_length=255, blank=True, null=True)
-    # on_time = models.BooleanField(default=True)
```
Comment (Menda) [inline workflow/models.py]: @glind is there a reason to keep them commented?

File: workflow/models.py (hunk 1 above)
Comment (jefmoura) [reply workflow/models.py]: Better than `number`

**Comment by Menda** [comment]
@jefmoura I talked with Greg and double checked them in the code. There is no problem to delete them. Now even less that we are split from CE

**Pull request merged**